### PR TITLE
Add CORS origin * to POST /contracts/call

### PR DESCRIPTION
**Pull request by leric7**
**Description**:

Add CORS origin * support to `POST /contracts/call` endpoint.
Motivation: Making POST API call from browser errors out with CORS preflight check failed.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/controller/ContractController.java [hunk 1]
```diff
@@ -63,6 +64,7 @@ class ContractController {
     private final ContractCallService contractCallService;
     private final Bucket bucket;
 
+    @CrossOrigin(origins = "*")
```
Comment (steven-sheehy) [inline hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/controller/ContractController.java:67]: This needs a test in `ContractControllerTest` that sets an `Origin: example.com` header and verifies the `Access-Control-Allow-Origin: *` in the response.

**Review by steven-sheehy — COMMENTED** [review]
Thanks for your contribution. It seems our load balancer and/or our proxy in public environment automatically adds the `Access-Control-Allow-Origin: *` necessary so that calling `/api/v1/contracts/call` via our [Swagger](https://mainnet-public.mirrornode.hedera.com/api/v1/docs/#/contracts/contractsCall) page works fine. But if you run it locally it is missing this header so it's a good addition.

**Review by jascks — DISMISSED** [review]
LGTM, the proposed test withstanding...

**Review by steven-sheehy — DISMISSED** [review]
LGTM thanks!

**Comment by steven-sheehy** [comment]
Looks like the test case fails. Can you check?

**Comment by leric7** [comment]
@steven-sheehy , fixed the test case failing.

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**